devel/fortio/http.go

```diff
@@ -710,3 +711,69 @@ func DynamicHTTPServer() int {
 	}(port)
 	return port
 }
+
+// DebugHandler returns debug/useful info
+func DebugHandler(w http.ResponseWriter, r *http.Request) {
+	LogVf("%v %v %v %v", r.Method, r.URL, r.Proto, r.RemoteAddr)
+	var buf bytes.Buffer
+	buf.WriteString("Φορτίο version ")
```

Comment:
I think this would be a lot easier/more idiomatic using a `text/template`. Something like:

```go
var debugTemplate = `
Φορτίο version: {{version}}
 echo debug server on {{hostname}}
 - request from {{r.RemoteAddr}} {{r.Method}} {{r.URL}} {{r.proto}}
   headers:
   {{ range name, vals := r.Header }}
        {{name}}: {{range vals}} {{ val }},{{end}}
   {{end}}
...etc
`

func DebugHandler(w http.ResponseWriter, r *http.Request) {
   ...
   err := template.Execute(w, debugTemplate, r, other, args, we, need)
   ...
}
```